ui/app/models/variable.js

```diff
@@ -67,6 +67,6 @@ export default class VariableModel extends Model {
    */
   setAndTrimPath() {
     this.set('path', trimPath([this.path]));
-    this.id = this.path;
+    this.set('id', this.get('path'));
```

Comment:
`nit` `non-blocking`:  you could have this be `setProperties`.

https://api.emberjs.com/ember/4.4/functions/@ember%2Fobject/setProperties

Reply:
I think that would cause an error; I want to set the `id` only after I set the `path` above, and editing both in a single `setProperties` would not give me a modified path.

Reply:
Gotcha! We should make sure `setAndTrimPath` doesn't run on `edit` and only on `create` since `path` shouldn't be a mutable property.